docs/aerodromes/classc/Darwin.md

```diff
@@ -47,6 +47,45 @@ d) Runway 11 and Runway 36;
 e) Runway 11 and Runway 18;  
 f) Runway 11 only.  
 
+## LAHSO
+!!! warning "Important"
+    Due to its operational complexity, LAHSO **must be authorised by a member of the VATPAC ATS or Events Staff Team or a C3-rated controller**.
+
+    Very little benefit is achieved by running LAHSO without a flow controller and it is almost exclusively reserved for our busiest events.
+
+LAHSO is the independent operation of two crossing runways for arrivals and departures. Whilst arrivals to each runway are considered independent, they will still be flowed to provide some degree of lateral segregation. ADC must maintain a visual separation standard between both aircraft during their approach and in the event of a go around.
+
+ADC must provide traffic information to both aircraft where a LAHSO standard has been applied between them. Additionally, the active LAHSO aircraft (the aircraft landing with a requirement to not pass the crossing runway) must be explicitely instructed to hold short of the crossing runway. Passive aircraft (the aircraft with full use of their assigned runway) must be informed that the active aircraft will hold short.
+
+### Weather Conditions
+LAHSO may only be used where the weather conditions meet the following minimum requirements:
+
+| Element | Criteria |
+| ------- | -------- |
+| Cloud Ceiling | `A030` or higher |
+| Visibility | 8km or greater |
+| Surface Condition | Dry |
+| Windshear | None Reported |
+
+Two simultaneous landings may be conducted by both day and night. A simultaneous takeoff and landing may only be conducted by day.
+
+### Runway Mode
+The only approved LAHSO mode at YPDN is **36A/29AD**. Runway 36 is considered the active runway, while runway 29 is considered the passive runway.
+
+!!! phraseology
+    **DN ADC:** "YZU, traffic is an E120 landing on the crossing runway, hold short runway 29, cleared to land runway 36."  
+    **YZU:** "Hold short runway 29, cleared to land runway 36, YZU"
+
+    **DN ADC:** "ANO407, traffic is a Seminole landing on the crossing runway, will hold short. Runway 29 cleared to land"  
+    **ANO407:** "Runway 29 cleared to land, AN407"
+
+### Go arounds
+To prepare for a go around of one or both aircraft, ADC shall monitor both flight paths and provide speed adjustments where necessary to ensure that in the event of a go around, the aircraft remain clear of each other. ADC must maintain a visual separation standard between both aircraft until a radar standard (3nm or 1000ft) is available, then the aircraft may be transferred to the TCU controller.
```

Comment:
Do you commonly get missed approach info in case of go around? Eg "in case of missed approach, passing 500 feet, turn left heading 150, climb and maintain 5000" or something like that? Maybe a tip?

Reply:
Not that I'm aware of... might create an expectation bias if instruction changes late and under pressure (compromised Sep scenario)

@mattk04 heard any different?

Reply:
Nah, never had that for LAHSO. The only time I've had an amended missed approach procedure expectation is when we've requested it proactively.